projects/packages/sync/src/replicastore/class-table-checksum.php

```diff
@@ -304,6 +304,14 @@ protected function get_default_tables() {
 				'table_join_field'          => 'order_item_id',
 				'is_table_enabled_callback' => array( $this, 'enable_woocommerce_tables' ),
 			),
+			'wc_orders'                  => array(
+				'table'                     => "{$wpdb->prefix}wc_orders",
+				'range_field'               => 'id',
+				'key_fields'                => array( 'id', 'parent_order_id' ),
+				'checksum_text_fields'      => array( 'type', 'billing_email', 'payment_method_title', 'transaction_id', 'ip_address', 'customer_note' ),
```

Comment:
Checksums should be lightweight, are all these columns critical? Without looking into all the HPOS data I would lean towards ip_address and customer_note being not included.